build/compile.mk

```diff
@@ -6,7 +6,7 @@ GO         ?= go
 BUILD_DIR  ?= ./bin
 PROJECT_MODULE ?= $(shell $(GO) list -m)
 # $b replaced by the binary name in the compile loop, -s/w remove debug symbols
-LDFLAGS    ?= "-s -w -X main.version=$(PROJECT_VER) -X main.appName=$$b -X $(PROJECT_MODULE)/internal/client.version=$(PROJECT_VER)"
+LDFLAGS    ?= "-s -w -X main.version=$(PROJECT_VER) -X main.appName=$$b -X $(PROJECT_MODULE)/internal/client.version=$(PROJECT_VER) -X $(PROJECT_MODULE)/internal/cli.Version=$(PROJECT_VER)"
```

Comment:
`main.version` and `client.version` should be able to reference `cli.Version` now since we export it from the package